02-spatial-data.Rmd

```diff
@@ -212,29 +224,33 @@ For more advanced map making we recommend using a dedicated visualization packag
 - map export 
 -->
 
-<!-- Maybe show also somewhere that `world[0]` produces only a plot of the geometry which is rather useful if you do not want to plot a specific attribute. This way, you can for example dismiss the col = "white"-argument in your Nigeria example. -->
+<!-- Maybe show also somewhere that `world[0]` produces only a plot of the geometry which is rather useful if you do not want to plot a specific attribute. This way, you can for example dismiss the col = "white"-argument in your Nigeria example.
+Sorry for commenting on this again but just to clarify africa[0] selects zero columns but since the geometry column is sticky it won't be dismissed. Neverthess, to be more explicit one should probably use plot(st_geometry(africa))-->
 
 ```{r, eval=FALSE, echo=FALSE}
 plot(africa[0], lwd = 3, main = "Nigeria in context", border = "lightgrey")
 ```
 
 #### Further work
 
-**sf** makes R data objects more closely aligned to the data model used in GDAL and GEOS, in theory making spatial data operations faster.
-The work here provides a taste of the way **sf** operates but there is much more to learn (see Chapter \@ref(spatial-data-operations)). And there is also a wealth of information in the highly recommended vignettes of the package.
+**sf** makes R data objects more closely aligned to the data model used in GDAL and GEOS, in theory making spatial data operations faster <!--I am not sure what you mean-->.
+The work here provides a taste of the way **sf** operates but there is much more to learn (see Chapter \@ref(spatial-data-operations)).
+And there is also a wealth of information in the highly recommended vignettes of the package.
 
-As a final exercise, we'll see one way of how to do a spatial overlay in **sf**. First, we convert the countries of the world into centroids, and then subset those in Asia. Finally, the `summary`-command tells us how many centroids (countries) are part of Asia (43) and how many are not (134).
+As a final exercise, we will see one way of how to do a spatial overlay in **sf**.
+First, we convert the countries of the world into centroids, and then subset those in Asia. Finally, the `summary`-command tells us how many centroids (countries) are part of Asia (43) and how many are not (134).
 
 ```{r}
 world_centroids = st_centroid(world)
 sel_asia = st_intersects(world_centroids, asia, sparse = FALSE)
 summary(sel_asia)
 ```
 
-Note: another way of achieving the same result is with a GEOS function for identifying spatial overlay, which we'll cover in more detail in Chapter \@ref(spatial-data-operations).
+<!--Note: another way of achieving the same result is with a GEOS function for identifying spatial overlay, which we will cover in more detail in Chapter \@ref(spatial-data-operations).-->
```

Comment:
Fair enough. Do you have a suggestion on the ideal solution for that? Feel free to update the content to represent that ideal world - I think it's useful to say that sometimes you can achieve the same result using a spatial operation (using GEOS in the background as you say - happy to spell that out or just not mention GEOS as you prefer) *and* an attribute operation.